### DISCUSSION TITLE
Wrappers and Rest API

### DISCUSSION THREAD

**Discussion by governoze** [supabase/supabase#15866] [Questions]
Can I use wrappers with a rest api?

**Comment by GaryAustin1** [comment]
I would think you could query the wrapper table with the standard database REST API as long as you put in a schema available to the API like public.

Edit... I am now thinking you are asking if you can use wrappers to access a REST API.  I don't think that is supported yet.

**Reply by governoze** [reply]
yes exactly, do you plan to make REST APIs available on the wrappers? Thanks

**Reply by GaryAustin1** [reply]
I believe their plan is to keep expanding wrappers, but I am not an employee, and don't have internal details.  They do invite contributions here: https://supabase.github.io/wrappers/contributing/ .
Blog with more details:
https://supabase.com/blog/postgres-foreign-data-wrappers-rust 
Maybe a dev will comment.
